packages/teleport/src/services/apps/apps.ts

```diff
@@ -39,11 +39,19 @@ const service = {
         value: json.value as string,
       }));
   },
+
+  getAppFqdn(params: AppParams) {
+    return api
+      .get(cfg.getAapFqdnUrl(params))
+      .then(json => ({
+        fqdn: json.fqdn as string,
+      }));
+  },
 };
 
 export default service;
 
-type CreateAppSessionParams = {
+type AppParams = {
```

Comment:
Since we exported the `UrlAppParams` defined in our `config.ts`, we can import it here and use that instead and avoid dup type